shell.nix

```diff
@@ -19,6 +19,7 @@ pkgs.mkShell {
     hinotify
 
     # Haskell
+    unstable.ghc
```

Comment:
Interesting. Why?

Reply:
> Interesting. Why?

oh whoops. I was playing with `dante` (for emacs) which tries to launch`ghci` in a pure nix shell.  that doesn't need to be here.

Reply:
Yeah, I like `dante` (and Intero before it).

I tend to use [`ghcid`](https://github.com/ndmitchell/ghcid) these days, which continually checks your code and displays compile errors in its own window (which is _amazing_ when refactoring across modules). It's getting stuck on the Template Haskell that grabs the version. Works when compiling, but not in ghcid 🤷‍♀️ The solution is splitting that module out into the executable instead of the library, but just haven't done that yet.